### ISSUE TITLE
[Dribbblish] Totally broken after spotify update

### ISSUE THREAD

**Issue by rolnix45** [spicetify/spicetify-themes#934]
![image](https://github.com/spicetify/spicetify-themes/assets/96535686/5d66144b-2cd2-4f32-9d5e-6799d97e32ef)

Spotify for Windows
1.2.12.902.g1924b088
Spicetify v2.18.1

**Comment by Potato22** [comment]
Additional: Happens on all variant
![image](https://github.com/spicetify/spicetify-themes/assets/39096741/dd34d6d0-d870-47e0-ac09-6085bd617d04)

**Comment by RMD-A** [comment]
same for me

**Comment by Kattuuss** [comment]
same for me too

**Comment by christhoperca** [comment]
Same problem as well
![image](https://github.com/spicetify/spicetify-themes/assets/130477138/7359dc36-acd4-419b-a1c9-bc75ef61b748)

**Comment by charagender** [comment]
Same here

![image](https://github.com/spicetify/spicetify-themes/assets/129703415/476e6eca-f01c-4745-84d7-0269dd202e26)

**Comment by NaufalF22** [comment]
![gambar](https://github.com/spicetify/spicetify-themes/assets/63091744/2b1e0dcc-8be8-4c8a-9231-40012c36217c)
Same

**Comment by kaiyin2** [comment]
In the meantime, you can downgrade to Spotify version 1.2.10.760 here: https://spotify.en.uptodown.com/windows/download/101586881

Everything should work for the time being until the pull request is reviewed and merged and the problem is fixed.

Make sure that after you install the previous version of Spotify you run:
```
spicetify backup
spicetify apply
```

**Comment by harbassan** [comment]
should be working now

**Comment by charagender** [comment]
dribblish was working again ... until I updated spicetify, which seems to have broken it again

![image](https://github.com/spicetify/spicetify-themes/assets/129703415/54e881a0-22bd-43a6-84cd-7e136454c3a2)

**Comment by Bumblebytrox** [comment]
Yea dribblish is broken again.

**Comment by carlocastoldi** [comment]
That PR did not fix the issue for me

**Comment by RMD-A** [comment]
its working now :)

**Comment by personinblack** [comment]
Am I doing something wrong? (spotify version: "1.2.11.916.geb595a67", spicetify version: "spicetify-cli-2.20.1")
![???](https://github.com/spicetify/spicetify-themes/assets/24732321/8c41f01a-0e50-419c-8e73-8941b295f22b)

**Comment by harbassan** [comment]
for the time being turn off ylx in experimental features.

**Comment by personinblack** [comment]
Sorry, that did not fix the problem for me.

**Comment by personinblack** [comment]
With the new Spotify update (1.2.13.661.ga588f749) it is fixed.

**Comment by harbassan** [comment]
yeah i just fixed it. should be working flawlessly now except for folder images on ylx (might work on that sometime in the future)
